app/presenters/saml_request_presenter.rb

```diff
@@ -0,0 +1,64 @@
+class SamlRequestPresenter
```

Comment:
If this is a presenter, should we rename `OpenidConnectAuthorizeDecorator` to be a presenter to match?